libvmaf/meson_options.txt

```diff
@@ -27,3 +27,13 @@ option('enable_float',
     type: 'boolean',
     value: false,
     description: 'Compile floating-point feature extractors into the library')
+
+option('enable_cuda',
+    type: 'boolean',
+    value: true,
+    description: 'Enable CUDA support')
+
+option('enable_nvtx',
+    type: 'boolean',
+    value: false,
+    description: 'Enable NVTX range support')
```

Comment:
newline

Reply:
I think this is already resolved by Kyle.